Makefile.am

```diff
@@ -60,6 +60,21 @@ GIRS =
 TYPELIBS = $(GIRS:.gir=.typelib)
 endif
 
+# These bits based on gnome:librsvg/Makefile.am
+if ENABLE_RUST
+if RUST_DEBUG
+CARGO_RELEASE_ARGS=
+else
+CARGO_RELEASE_ARGS=--release
+endif
+
+check-local:
+	cd $(srcdir)/rust && CARGO_TARGET_DIR=@abs_top_builddir@/target cargo test
```

Comment:
Do we actually need this target? Or are we planning to use the rust testing framework as well? Might want to rename it to `rust-check` or similar.